src/data/communities.json

```diff
@@ -134,5 +134,15 @@
     "instagram": "https://www.instagram.com/build2learn_c",
     "linkedin": "https://www.linkedin.com/company/build2learn/",
     "github": "https://github.com/build2learn-in/"
+  },
+  {
+    "name": "Chennai Web Engineering",
+    "logo": "https://yt3.googleusercontent.com/fsQ4GZ83_gLntVj3-TbGeM5g6e2ojYFwW4CPkMG-1F9bdfgFJHAY5mkKQngW2soYITIXcnFU=s160-c-k-c0x00ffffff-no-rj",
```

Comment:
Hey @MeiyappanKannappa 
Thanks a lot for the contribution, 
Great to see Chennai Web Engineering be part of tamilnadu.tech

I noticed, you had added the logo from a domain which is not currently configured in our next.config.ts 

I would kindly suggest you to use imgbb and get an embeddable link similar to the other communities added in the same file. 
or as an alternative since meetup cdn is added to our next config file you can also replace this with the image url from your meetup group :)

by doing this, it would help us to display the image along side the event, 
thank you so much for understanding.

Cheers :)